src/main/java/edu/cornell/kfs/concur/businessobjects/ConcurAccountInfo.java

```diff
@@ -9,6 +9,11 @@
     protected String objectCode;
     protected String subObjectCode;
     protected String projectCode;
+    protected String orgRefId;
+    
+    public ConcurAccountInfo() {
```

Comment:
I wonder if it would make sense to add another constructor that also takes in the orgRefId as a param.